### ISSUE TITLE
FS15 Holmer DLC incompatability

### ISSUE THREAD

**Issue by JDTiger** [Courseplay/courseplay#1297]
The Holmer DLC is finally out, but Courseplay doesn't recognize the overloader implements for the Terra Variant as overloaders for overloading modus...
would be nice to have that fixed :D

**Comment by SnoopBok** [comment]
Indeed, I think it's a nice DLC. I hope you guys will support it :-)

**Comment by ThomasGaertner** [comment]
yea , I'm working on it ...

**Comment by Stolli1996** [comment]
Get also supported the diffrent Steering modes?

**Comment by ThomasGaertner** [comment]
at the moment I'm fighting with game crashes when testing the overloader functions. Did you guys observe any crashes during playing with the Holmer DLC without CP ?

**Comment by skydanc3r** [comment]
i have no crashes

**Comment by SnoopBok** [comment]
I have nog crashes at all.

**Comment by PostMan1001** [comment]
No crashes without courseplay

**Comment by PostMan1001** [comment]
It will be great to have this update to course play

**Comment by b101uk** [comment]
No crashes here using the Terra Variant overloader functions, have even used Courseplay with the Zunhammer slurry equipment as an overloader into the FRC tank (though you have to manually press the overloader key rather than Courseplay doing it automatically)

there is also the problem of the Terra Variant getting stuck at 19km/h after its been on a CP route if you try to drive it afterwards (resetting it to the farm fixes the problem, as dose restarting the game)

**Comment by ThomasGaertner** [comment]
@b101uk : I can confirm this behavior. my current points are:

- got stuck on 19km/h 
- sugarbeet harvester got stuck in a undefined crab steering position after unloading to a trailer
- fill trigger on the slurry overloader will not be recognised
already running: 
harvesting surgarbeet, overloading sugarbeet, manure spreading and loading with CP,slurry spreading and refilling.

**Comment by Laui51** [comment]
Nice to hear! Hopefully GIANTS can fix the bugs quickly.

Thanks again for your great work!

**Comment by AlexStephenLytton** [comment]
I also can confirm the 19 km/h Problem ... but no error in console...

**Comment by ThomasGaertner** [comment]
please test v.115 intensely . To activate the crab steering, just set the crab steering mode before starting courseplay. The side will be switched concerning the fruit area. try out :-)
I reccommend to activate pathfinding on the modes 2 vehicles.

**Comment by PostMan1001** [comment]
How do we download V115

**Comment by JDTiger** [comment]
PLS remind me: what was pathfinding for &how do i activate it?
must have missed that feature....

**Comment by ThomasGaertner** [comment]
basicly it was made to prevent the tractor in combined mode to go through the fruit while crossing the field for unloading. at the moment it only prevents the tractor for unloading the combine while the pipe is on the fruit side.

**Comment by JDTiger** [comment]
ah ok... well translated feature :D 
Germanversion of CP: fruchtumfahrung (direct translation drive around fruit)
Englishversion of CP: pathfinding (direct translation wegfindung)

makes no real sense for me as german but ok :D

**Comment by Stolli1996** [comment]
Hi
So i have the new update and now my log gets very big over 3 mb
and the most enrtys are 
distance: 4.216719; backmarker -5.974976; y:18.926 vs Height:18.547
distance: 4.099568; backmarker -5.974976; y:18.929 vs Height:18.547
distance: 3.981336; backmarker -5.974976; y:18.934 vs Height:18.547
distance: 3.866885; backmarker -5.974976; y:18.939 vs Height:18.547
distance: 3.754209; backmarker -5.974976; y:18.941 vs Height:18.547
distance: 3.638269; backmarker -5.974976; y:18.943 vs Height:18.547
distance: 3.523035; backmarker -5.974976; y:18.944 vs Height:18.547
distance: 3.402061; backmarker -5.974976; y:18.943 vs Height:18.547
distance: 3.283762; backmarker -5.974976; y:18.943 vs Height:18.547

what does it mean and must this written in the log
i think its entrys about the ridgemakers

MFG

**Comment by ThomasGaertner** [comment]
@Stolli1996 : frogotten print, fixed in v.116...
@JDTiger : it makes sense,because  it's both : it should be a external function to find a path arround the fruit. but it's not worked out yet. I will check the slurry loading thing.it worked with my tests...

**Comment by Jumurdzsak** [comment]
I have tried to use sowing machine "horsch maestro 12" hooked after the tera variant with courseplay.
Chosen "fieldwork" and generated a path. But after pressing the drive course my game window becomes unresponsive (until closing the whole game window), but the sound stays on.
Steering, game menus, hotkeys, stop driver function also not working in this status.

Anyone else had this issue?

**Comment by LordNighthawk** [comment]
I have the same problem when i tried to use the Horsch Tiger 10LT. Game freezes and has to be terminated with the task manager.
Log gets flooded with : 

C:/Users/XXXXXX/Documents/my games/FarmingSimulator2015/mods//ZZZ_courseplay/specialTools.lua:515: attempt to index local 'tractor' (a nil value)

**Comment by LordNighthawk** [comment]
Ok, tested with Tiger 10LT and Horsch Maestro 12 and no recognizable errors yet. Seems to work fine now.

**Comment by LordNighthawk** [comment]
Still having the problem that the Terra Variant drives a far too tight curve in the following scenarion.
Harvester is harvesting a lane and facing the direction where the Terra Variant is waiting (means he moves up to him). Harvester is full and stops, Terra Variant moves up to him for to unload, makes a turn around behind him to the unload side but when he tries to get beside the harvester the curve is too tight and he gets stuck with the distance to the harvesters cutter (stuck in traffic). This happens in 90% of the cases with 4WD and in ~25% with only front wheel steering mode.

**Comment by FSFiestel** [comment]
I'm using .0119 and am having the issue that the Terra Dos T4-40 harvester gets stuck in the crab position after unloading to the Terra Variant.  In all tests I am driving the Terra Dos T4-40.  I've tried numerous settings to try and pinpoint the cause, sometimes it works fine and then all of a sudden gets stuck in the crab position.  One time I had the Terra Variant offset at the default of 6.5 mtrs which won't allow it to drive alongside the Terra Dos T4-40 in the crab left position (I find that 7 to 7.5 mtrs offset will allow the Terra Variant to pull alongside ok).  I had the pipe already extended on the Terra Dos T4-40 and then returned the steering to normal.  The Terra Variant started to pull alongside while the Terra Dos T4-40 was still straightening out and as soon as it started to unload into the Terra Variant, the steering on the Terra Dos T4-40 stopped working and was then stuck in a partial crab position.  It appears to be an issue with the exact moment unloading starts, but I can't seem to pinpoint why one time the steering will continue to work properly and then the next will get stuck in the crab position.  The really odd thing is that I harvested Sosnovka field 25 with no problem and am now having the problem while harvesting field 23.

**Comment by WolfWarrior619** [comment]
I'm new to GitHub so bear with me. Im using the Terra Variant on developer's version of Courseplay and I still can't get the Overloader function to respond. I'm still getting the error, "wrong trailer". Any help would be appreciated. Cheers!

**Comment by jor52** [comment]
Is there any news on this problem? did i mis the solution?

**Comment by ThomasGaertner** [comment]
#1346
